pulp_container/app/registry_api.py

```diff
@@ -699,7 +699,8 @@ def put(self, request, path, pk=None):
 
             dispatched_task = dispatch(
                 add_and_remove,
-                [f"upload:{pk}", repository],
+                shared_resources=[f"upload:{pk}"],
```

Comment:
Shared is ok, since we really only _use_ this to identify the task.
(Reminds me that we wanted to find a way to run tasks in the foreground [not dispatching them].)